BuildResidentialHPXML/measure.rb

```diff
@@ -1025,18 +1029,17 @@ def arguments(model)
     arg.setDefaultValue(HPXML::HVACTypeCentralAirConditioner)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('cooling_system_cooling_efficiency_seer', true)
-    arg.setDisplayName('Cooling System: Rated SEER')
-    arg.setUnits('SEER')
-    arg.setDescription("The rated efficiency value of the #{HPXML::HVACTypeCentralAirConditioner} cooling system.")
-    arg.setDefaultValue(13.0)
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('cooling_system_cooling_efficiency_type', cooling_efficiency_type_choices, true)
+    arg.setDisplayName('Cooling System: Efficiency Type')
+    arg.setDescription('The efficiency type of the cooling system.')
+    arg.setDefaultValue(HPXML::UnitsSEER)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('cooling_system_cooling_efficiency_eer', true)
-    arg.setDisplayName('Cooling System: Rated EER')
-    arg.setUnits('EER')
-    arg.setDescription("The rated efficiency value of the #{HPXML::HVACTypeRoomAirConditioner} cooling system.")
-    arg.setDefaultValue(8.5)
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('cooling_system_cooling_efficiency', true)
+    arg.setDisplayName('Cooling System: Efficiency')
+    arg.setUnits("#{HPXML::UnitsSEER} or #{HPXML::UnitsEER}")
+    arg.setDescription("The rated efficiency value of the cooling system. System types #{HPXML::HVACTypeCentralAirConditioner} and #{HPXML::HVACTypeMiniSplitAirConditioner} use #{HPXML::UnitsSEER}. System type #{HPXML::HVACTypeRoomAirConditioner} uses #{HPXML::UnitsEER}.")
```

Comment:
The description of which system types use which efficiency metrics should really go with the above argument, not this one. Same comment presumably applies to some arguments below.

Also, it's probably confusing to a user that it doesn't say anything about evaporative coolers. We should say that these arguments are ignored for them. (Alternatively we could make the arguments optional, but that makes me a bit nervous.)